test/UnitPasswordProtected.cpp

```diff
@@ -64,6 +64,9 @@ UnitBase::TestResult UnitPasswordProtected::testPasswordProtectedDocumentWithout
         COOLProtocol::getTokenString(tokens[2], "kind", errorKind);
         LOK_ASSERT_EQUAL(std::string("load"), errorCommand);
         LOK_ASSERT_EQUAL(std::string("passwordrequired:to-view"), errorKind);
+
+        response = helpers::getResponseString(socket, "error:", testname);
+        LOK_ASSERT_MESSAGE("Unexpected second error message", !response.size());
```

Comment:
Perhaps we could also compare the full error message we expect? Because in some cases there could be a different failure, which should break this test.